nrf_modem/doc/CHANGELOG.rst

```diff
@@ -12,20 +12,21 @@ All notable changes to this project are documented in this file.
 nrf_modem 2.0.0
 ***************
 
+* Added modem fault reasons to the :file:`nrf_modem.h` file.
+* Added :c:func:`nrf_modem_is_initialized` function to query the modem initialization status.
+* Added :c:func:`nrf_modem_os_event_notify` function to wake up threads sleeping in the :c:func:`nrf_modem_os_timedwait` function.
+* Added :c:func:`nrf_modem_os_sem_count_get` function to retrieve a semaphore's count.
+* Added :c:func:`nrf_modem_os_trace_alloc` and :c:func:`nrf_modem_os_trace_free` functions to allocate trace metadata on a dedicated memory heap.
 * Numerous fixes and improvements to networking sockets.
 * Increased logging output (in log version of the library).
-* Improved modem fault handling. A new field has been added to :c:type:`nrf_modem_init_params_t` to receive a callback upon modem faults.
-* Updated :c:func:`nrf_modem_shutdown` to shutdown quicker when a debugger is attached or the modem has faulted.
-* Updated :c:func:`nrf_modem_os_timedwait`to return negative values, aligning with other APIs.
-* Fixed a bug which could lead to ``NRF_MODEM_GNSS_EVT_FIX`` being sent before ``NRF_MODEM_GNSS_EVT_UNBLOCKED``.
-* Added modem fault reasons to :file:`nrf_modem.h`.
-* Added :c:func:`nrf_modem_is_initialized` to query the modem initialization status.
-* Added :c:func:`nrf_modem_os_event_notify` to wake up threads sleeping in :c:func:`nrf_modem_os_timedwait`.
-* Added :c:func:`nrf_modem_os_sem_count_get` to retrieve a semaphore's count.
-* Added :c:func:`nrf_modem_os_trace_alloc` and :c:func:`nrf_modem_os_trace_free` to allocate trace metadata on a dedicated memory heap .
-* Renamed ``NRF_SO_HOSTNAME`` to ``NRF_SO_SEC_HOSTNAME``.
-* Renamed ``NRF_SO_CIPHERSUITE_LIST`` to ``NRF_SO_SEC_CIPHERSUITE_LIST``.
-* Renamed ``NRF_SO_CIPHER_IN_USE`` to ``NRF_SO_SEC_CIPHER_IN_USE``.
+* Improved modem fault handling.
```

Comment:
These three entries were at the top because they are the highlights of this release..
is there a way we can highlight them still?

Reply:
There is no other way to highlight them. I have moved these entries to the top.